### PR TITLE
Update content/zh/docs/concepts/services-networking/network-policies.md

### PR DESCRIPTION
**Pull request by ydcool**
Update zh docs for release-1.18

### CODE REVIEW

**Comment by tengqm** [comment]
Awesome!
/lgtm

File: content/zh/docs/concepts/services-networking/network-policies.md [hunk 1]
```diff
@@ -4,19 +4,31 @@ content_template: templates/concept
 weight: 50
 ---
 
+<!--
```
Comment (tanjunchen) [inline content/zh/docs/concepts/services-networking/network-policies.md:7]: /LGTM
Thanks

**Pull request merged**